packages/devtools_extensions/extension_config_spec.md

```diff
@@ -0,0 +1,13 @@
+The `config.yaml` file for a DevTools extension must follow the format below.
+
```

Comment:
How about naming it "extension_config.yaml"? 
There can be other things to configure, right?

Reply:
The name of this file is set. This is plumbed through the DevTools extension framework and `package:extension_discovery`. This markdown file is just to document the expected fields for the file.

Reply:
the file lives at:
```
extension/
  devtools/
    config.yaml
```
so the fact that it is a configuration for a devtools extension is implicit in the path